### PR TITLE
Update to help docs 

### PR DESCRIPTION
**Pull request by SydneyhSmith**
# PR Summary

This PR fixes schema errors in all of the markdown help files and updates the yaml help file.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [X] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

File: help/Find-PSResource.md [hunk 1]
```diff
@@ -31,21 +31,21 @@ Searches for packages from a repository (local or remote), based on `-Name` and
...
-[[Name <string>] [-Prerelease]  [-Type <Microsoft.PowerShell.PowerShellGet.UtilClasses.ResourceType[]>]
+[Name <string>] [-Prerelease]  [-Type <Microsoft.PowerShell.PowerShellGet.UtilClasses.ResourceType[]>]
 [-Repository <string[]>] [-Credential <pscredential>] [-IncludeDependencies] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 The `Find-PSResource` cmdlet searches for a package from a repository (local or remote) based on `-Name` or other package properties.
 
 ## EXAMPLES
-These examples assume that the PSGallery repository is registered and contains the packages we are searching for.
```
Comment (alerickson) [inline help/Find-PSResource.md:48]: Do we want to remove this?  Was this automatically removed from one of the PlatyPS cmdlets or is this a schema violation?

File: help/Get-PSResource.md [hunk 2]
```diff
@@ -150,39 +151,6 @@ Accept wildcard characters: False
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
-## OUTPUTS
```
Comment (alerickson) [inline help/Get-PSResource.md:153]: Similar question here, is `Outputs` something we don't want to include in the docs?

File: help/Install-PSResource.md [hunk 3]
```diff
@@ -78,26 +78,6 @@ PS C:\> Install-PSResource -RequiredResourceFile myRequiredModules.psd1
 ```
 
 Installs the PSResources specified in the psd1 file.
- 
- ### Example 5
```
Comment (alerickson) [inline help/Install-PSResource.md:82]: Is this a schema violation as well?

File: help/Find-PSResource.md (hunk 1 above)
Comment (SydneyhSmith) [reply help/Find-PSResource.md:48]: This is a schema violation, maybe I should move this description to the first example rather than delete completely

**Review by anamnavi — APPROVED** [review]
So we don't want `Output` section in the help docs? Also not sure what is happening in the xml file...it seems like we just added info for the ScriptFileInfo and other new cmdlets but erased other info? Maybe I misunderstood though, it was tough looking through the diff for that trying to understand the change made, what it should be, etc.

**Comment by SydneyhSmith** [comment]
> So we don't want `Output` section in the help docs? Also not sure what is happening in the xml file...it seems like we just added info for the ScriptFileInfo and other new cmdlets but erased other info? Maybe I misunderstood though, it was tough looking through the diff for that trying to understand the change made, what it should be, etc.

Thanks for catching the issue with the xml, just re-pushed it and seems right now...trying to figure out why all the Outputs gave schema errors

**Pull request merged**